src/integration-tests/test_restart_between_modes.py

```diff
@@ -561,10 +658,20 @@ def test_restart_between_non_FSM_and_FSM_unassign_queue(
     assignUnassignExistingQueues(existing_queues, consumer, leader)
 
 
-@pytest.fixture(params=[restart_cluster_as_fsm_mode, restart_cluster_as_legacy_mode])
+@pytest.fixture(
+    params=[
+        # (restart_as_fsm_mode, restart_as_legacy_mode),
+        # (restart_as_legacy_mode, restart_as_fsm_mode),
+        (restart_to_fsm_single_node_with_quorum_one, restart_as_legacy_mode),
+        # (
+        #     restart_to_fsm_single_node_with_quorum_one_and_start_others,
+        #     restart_as_legacy_mode,
+        # ),
```

Comment:
```suggestion
        (restart_as_fsm_mode, restart_as_legacy_mode),
        (restart_as_legacy_mode, restart_as_fsm_mode),
        (restart_to_fsm_single_node_with_quorum_one, restart_as_legacy_mode),
        (restart_to_fsm_single_node_with_quorum_one_and_start_others, restart_as_legacy_mode),
```
We do need to test all these scenarios

Reply:
Also, now in the [GitHub Actions UI](https://github.com/bloomberg/blazingmq/actions/runs/19330622351/job/55315803721?pr=1001), the test name shows as `test_restart_between_modes.py::test_restart_between_Legacy_and_FSM[single_node_fsm-strong_consistency-switch_cluster_mode0]`. `switch_cluster_mode0` is not very descriptive as we do not know what `0` means. Is there a way to name the test as `test_restart_between_modes.py::test_restart_between_Legacy_and_FSM[single_node_fsm-strong_consistency-restart_to_fsm_single_node_with_quorum_one]` for example?

You can ask @jll63 for help if needed